pages/index.tsx

```diff
@@ -29,27 +31,68 @@ const SignInScreen: React.FC = () => {
     }
   };
 
+  const toggleVisibility = () => {
+    inputType == "password" ? setInputType("text") : setInputType("password");
+  };
+
   return (
-    <form onSubmit={handleSubmit(handleSignin)}>
-      <div className={styles["container"]}>
-        <div className={styles["signin-container"]}>
-          <label>Email</label>
-          <input
-            id="email"
-            name="email"
-            {...register("email", { required: true })}
-          />
-          <label>Password</label>
-          <input
-            id="password"
-            name="password"
-            type="password"
-            {...register("password", { required: true })}
-          />
-          <Button type="submit">Sign In</Button>
+    <div className={styles["page-container"]}>
+      <form onSubmit={handleSubmit(handleSignin)}>
+        <Icon type="sundayfriendslogo" className={styles.SFlogo} />
+        <div className={styles["container"]}>
+          <div className={styles["box-styling"]}>
+            <div className={styles["signin-container"]}>
+              <h1 className={styles["welcome-title"]}>Welcome Back!</h1>
+              <h4 className={styles["title-subtext"]}>Sign in to continue</h4>
+              <div className={styles["input"]}>
+                <label className={styles["email-input-label"]}>EMAIL</label>
+                <div className={styles["input-field"]}>
+                  <input
+                    id="email"
+                    name="email"
+                    className={styles["email-input-field"]}
+                    {...register("email", { required: true })}
+                  />
+                </div>
+              </div>
+              <div className={styles["input"]}>
+                <label className={styles["password-input-label"]}>
+                  PASSWORD
+                </label>
+                <input
+                  id="password"
+                  name="password"
+                  type={inputType}
+                  className={styles["password-input-field"]}
+                  {...register("password", { required: true })}
+                />
+                <button
+                  className={styles["hide-password-button"]}
+                  onClick={toggleVisibility}
+                >
+                  <Icon type="hidepassword" className={styles.hidepassword} />
+                </button>
+              </div>
+              <div className={styles["forgot-password"]}>
+                {" "}
+                <a href="/">Forgot password? </a>
+              </div>
+              <div className={styles["bottom-row"]}>
+                <div>
+                  Haven&apos;t activated your account yet?{" "}
```

Comment:
nit: Could you use a string here for better readability? 
```suggestion
                 {"Haven't activated your account yet? "}
```